### ISSUE TITLE
Save global_keystore or ks to static memory

### ISSUE THREAD

**Issue by itisBaga** [ConsenSys/eth-lightwallet#149]
How ca I save ks object to use it after reloading page? I have treid any ways but ks lost prototype. Also I tried to stringify protorype, save it to indesedDB and localStorage, after that retrived prototype and set to ks object. Internal prototyp functions also lost. Is it possible to use library in this way or any ideas?

**Comment by christianlundkvist** [comment]
@itisBaga It sounds like you want to use the serialize function and save the serialized keystore to local storage:

https://github.com/ConsenSys/eth-lightwallet#keystoreserialize

**Comment by Priya-Nets** [comment]
How to use the saved serialised keystore for contract creation? I created a account using lightwallet and funded 1 ether on my private network. Now when I try to access the same keystore and deploy contract and set some values in contract doesn't seem to work. I cannot see values being set because when I try retrieving it, they hold the default values and not the updated one. Where am I going wrong?
@itisBaga @christianlundkvist

**Comment by itisBaga** [comment]
@Priya-Nets you should use serialize function, then seva kesytore to your storage, and after that you should get ks object and deserialize it with .deserialize() method. It was my mistake also) Good luck.

**Comment by sc0Vu** [comment]
@itisBaga Right now I am using data:uri and a link to let user download keystore json file, I wonder is there any better way to do that?
Thanks.

**Comment by itisBaga** [comment]
@sc0Vu I don't think so. You can generate file programmaticaly archive it and download like this, I think. Can I ask you, why you want let users to download ks object? Keeping phrase probably more user frendly way to backup walletm no?
